docs/core/porting/index.md

```diff
@@ -136,17 +136,17 @@ When porting your application to .NET, consider the following suggestions in ord
 
 ✔️ CONSIDER using the [.NET Upgrade Assistant](upgrade-assistant-overview.md) to migrate your projects. Even though this tool is in preview, it automates most of the manual steps detailed in this article and gives you a great starting point for continuing your migration path.
 
-✔️ CONSIDER examining your dependencies first. Your dependencies must target .NET 5, .NET Standard, or .NET Core.
+✔️ CONSIDER examining your dependencies first. Your dependencies must target .NET 6, .NET Standard, or .NET Core.
```

Comment:
This is about porting apps to .NET, if it's OK for dependencies to target .NET Core it must be OK to target .NET 5, which is later than Core.

```suggestion
✔️ CONSIDER examining your dependencies first. Your dependencies must target .NET 5+, .NET Standard, or .NET Core.
```